torch/_inductor/ir.py

```diff
@@ -3112,23 +3141,73 @@ def __init__(
             tuple(tensor_args),
             tuple(nontensor_args),
         )
+        self.use_cpp_op_schema = False
+
         if getattr(torch.ops.aten, kernel.__name__, None) is kernel:
             self.kernel = (
                 f"at::{kernel.__name__}"
                 if V.graph.cpp_wrapper
                 else f"aten.{kernel.__name__}"
             )
         else:
-            assert (
-                not V.graph.cpp_wrapper
-            ), f"{kernel.__name__} is not supported with cpp wrapper"
-            self.kernel = (
-                f"{kernel.__module__.replace('._ops.', '.ops.')}.{kernel.__name__}"
-            )
+            if V.graph.cpp_wrapper:
+                from torch._inductor.codegen.wrapper import (
+                    SUPPORTED_FALLBACK_CPP_WRAPPER,
```

Comment:
I feel you already have strict checking when getting the cpp kernel info. This supported list seems not necessary?

Reply:
Currently, I haven't found a full list of fallback OPs that are not exposed in `torch.ops.aten`, especially for the OPs that fall back implicitly. In this PR, I've only added UT for `repeat_interleave.Tensor` (the one that we've encountered in GPT-J). I plan to add more UTs in the follow-up PR and then remove this list. Does that sound good?

Reply:
Sure, sounds good to me.